mkdocs.yml

```diff
@@ -93,32 +93,32 @@ nav:
       - Provisional:
         - geps/gep-1324.md
         - geps/gep-1619.md
-        - geps/gep-1897.md
         - geps/gep-1867.md
+        - geps/gep-1897.md
       - Implementable:
-        - geps/gep-1742.md
-        - geps/gep-1686.md
         - geps/gep-1426.md
+        - geps/gep-1686.md
+        - geps/gep-1742.md
       - Experimental:
-        - geps/gep-1748.md
-        - geps/gep-1651.md
-        - geps/gep-1016.md
-        - geps/gep-957.md
         - geps/gep-713.md
+        - geps/gep-957.md
+        - geps/gep-1016.md
+        - geps/gep-1651.md
         - geps/gep-1709.md
+        - geps/gep-1748.md
         - geps/gep-2257.md
       - Standard:
-        - geps/gep-1364.md
-        - geps/gep-1323.md
-        - geps/gep-922.md
-        - geps/gep-917.md
-        - geps/gep-851.md
-        - geps/gep-820.md
-        - geps/gep-746.md
-        - geps/gep-726.md
-        - geps/gep-724.md
-        - geps/gep-718.md
         - geps/gep-709.md
```

Comment:
One nitpick: This was intentionally done in descending order so the most recent GEPs would be at the top of the list. I don't have a strong opinion here, and clearly other parts of this nav have it in ascending order so 🤷.

Reply:
I (eventually) decided that I'd prefer having them all ascending. Not a strong opinion here either, really -- anyone else feel strongly?

Reply:
I think consistency is better here. It's not a big enough deal to worry about too much - we can always change it later.